docs-rs/trait-tags.html

```diff
@@ -126,32 +126,39 @@
         return el;
     }
 
-    // For item listings in modules, sort the items by implemented Bevy
-    // traits and add corresponding tags.
-    // Send off all network requests simultaneously.
-    for (const kind of ['structs', 'enums', 'functions']) {
-        // Find the listing
-        let heading = document.getElementById(kind);
-        if (!heading) {
-            continue
-        }
-        const listing = heading.nextSibling;
-        const items = [];
-        for (const item of listing.children) {
-            // Only look at items, not their description
-            if (item.nodeName != "DT") {
+    const info_node = document.getElementById("bevy-traits-data");
+
+    if (info_node) {
+        const data_uri = info_node.href;
+        const base64_index = data_uri.indexOf(",") + 1;
+        const base64 = data_uri.substring(base64_index);
+        const json = atob(base64)
+        const info = JSON.parse(json);
+
+        // For item listings in modules, sort the items by implemented Bevy
+        // traits and add corresponding tags.
+        // Send off all network requests simultaneously.
```

Comment:
This doesn't send requests anymore
```suggestion
        // For item listings in modules, sort the items by implemented Bevy
        // traits and add corresponding tags.
```